scripts/qa_crowbarsetup.sh

```diff
@@ -2382,8 +2382,10 @@ function oncontroller_testsetup()
     fi
 
     #test for Glance scrubber service, added after bnc#930739
-    su - glance -s /bin/sh -c "/usr/bin/glance-scrubber --config-dir /etc/glance" \
-        || complain 113 "Glance scrubber doesn't work properly"
+    if iscloudver 6plus || [[ $cloudsource =~ develcloud ]]; then
```

Comment:
no, it does not work for GM5 or GM5+up. thats why it is checking for 6plus only.